Src/AutoFixture/RandomNumericSequenceGenerator.cs

```diff
@@ -53,7 +51,17 @@ public RandomNumericSequenceGenerator(params long[] limits)
 
             if (limits.Length < 2)
             {
-                throw new ArgumentException("The limit must be a sequence of two or more integers.");
+                throw new ArgumentException("limits must be two or more Int64.", "limits");

```

Comment:
Using *numbers* sounds good – Though, I think *numbers in ascending order* can be more precise :wink: